### ISSUE TITLE
Map can't be closed with Escape after using "Show on Map"

### ISSUE THREAD

**Issue by SeanSawyers** [Questie/Questie#2990]
Whenever I Right-Click on a quest on the Questie tracker and choose the option "Show on Map," while this does properly open the map and shows the objective, it also renders the map unable to be closed using the Escape button. Escape does still work normally otherwise though, as I can hear the Options menu being opened and closed behind it. The Keybind for opening/closing the map and the 'X' button in the top-right do both still work correctly. Using version 6.4.1.

**Comment by BreakBB** [comment]
Yeah that is a known issue (can't find it atm), but we haven't really looked into it much. Thanks for bringing it up again @SeanSawyers 👍🏻

**Comment by BreakBB** [comment]
I just noticed this is not a problem when using the map addon Leatrix Plus. Default map is still a problem.
